dwave/system/composites/cutoffcomposite.py

```diff
@@ -273,10 +273,12 @@ def sample_poly(self, poly, **kwargs):
         isolated = list(original.variables.difference(new.variables))
 
         if isolated and len(new) == 0:
-            # in this case all variables are isolated, so we just put one back
-            # to serve as the basis
-            term = isolated.pop(),
-            new[term] = original[term]
+            # in this case all variables are isolated, so find the variable with
+            # the strongest bias and use that as the seed for putting the other
+            # variables back in
+            v = max(isolated, key=lambda v: original.get((v,), 0.0))
+            isolated.remove(v)
+            new[(v,)] = original.get((v,), 0)
```

Comment:
Out of curiosity, what's the reason for not using `max(isolated, key=lambda v: original.get(v, 0.0))`  and `new[v] = original.get(v, 0)` ?

Reply:
terms are stored in frozensets cast from tuples. I think just sending `v` works but honestly I forget the actual rules for that...